tests/Tests/100__deploy/100__installation/102__post_install.robot

```diff
@@ -281,6 +281,22 @@ Verify RHODS Display Name and Version
     Should Be Equal       ${rhods_version_t[1]}   ${rhods_version}   msg=RHODS vesrion and label is not consistent
     Should Be Equal       ${rhods_displayname}   Red Hat OpenShift Data Science  msg=Dieplay name doesn't match
 
+Verify RHODS Notebooks Network Policies
+    [Documentation]    Verifies that the network policies for RHODS Notebooks are present on the cluster
+    [Tags]    Smoke    TESTME
```

Comment:
Can you replace TESTME for Tier1?

Reply:
waiting for polarion IDs. I would question also Smoke vs Sanity. WDYT? cc:@lugi0

Reply:
I think this test should run in Smoke.
I haven't been able to access Polarion for the whole afternoon, will provide IDs ASAP

Reply:
ODS-2045

Reply:
done